CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix inconsistent request body closing in `go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploghttp`. (#5954)
+- Fix inconsistent request body closing in `go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp`. (#5954)
```

Comment:
I don't know if this one (otlptracehttp) needs a changelog entry. It removes a duplicate body close, but there's no behavior change.

Reply:
I think that there can be a change as the first body close in `otlptracehttp` may drop the error

From https://pkg.go.dev/io#Closer

> The behavior of Close after the first call is undefined.